gardener-addon/files/certificate.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -e
+
+echo "---> Get Shoot Domain"
+SHOOT_DOMAIN="$(kubectl -n kube-system get configmap shoot-info -o jsonpath='{.data.domain}')"
+DOMAIN="$SUBDOMAIN.$SHOOT_DOMAIN"
+
+echo "---> Requesting certificate for domain ${DOMAIN}"
+cat <<EOF | kubectl apply -f -
+---
+apiVersion: cert.gardener.cloud/v1alpha1
+kind: Certificate
+metadata:
+  name: kyma-cert
+  namespace: kyma-installer
+spec:
+  commonName: "*.$DOMAIN"
+EOF
+
+while :
+do
+STATUS="$(kubectl get -n kyma-installer certificate.cert.gardener.cloud kyma-cert -o jsonpath='{.status.state}')"
+if [ "$STATUS" = "Ready" ]; then
+    break
+else
+    echo "Waiting for Certicate generation, status is ${STATUS}"
+    sleep $DELAY
```

Comment:
Envs should be quoted, also in other places

```suggestion
    sleep "${DELAY}"
```